src/getHumanAge.test.js

```diff
@@ -7,4 +7,44 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array with 2 integers', () => {
+    const result = getHumanAge(19, 19);
+
+    expect(Array.isArray(result)).toBe(true);
+    expect(result.length).toBe(2);
+
+    for (const value of result) {
+      expect(Number.isInteger(value)).toBe(true);
+    }
+  });
+
+  test('should return [0, 0] when both parameters are 0', () => {
+    expect(getHumanAge(0, 0)
+    ).toEqual([0, 0]);
+  });
+
+  test('should ignore decimal part of the input', () => {
+    expect(getHumanAge(15.7, 15.9)).toEqual([1, 1]);
```

Comment:
The test case for ignoring the decimal part of the input (`getHumanAge(15.7, 15.9)`) might not be necessary if the function is expected to handle only integer inputs, as per the task description. Consider whether this test aligns with the intended functionality of `getHumanAge`.